src/pages/_assets/welcome-content.tsx

```diff
@@ -0,0 +1,263 @@
+import { CardItem } from "../../components/WelcomeCard/welcome-card"
+
+// Section 1: Identity Management
+export const identityManagement: {
+  id: string
+  title: string
+  description: string
+  cards: CardItem[]
+} = {
+  id: "identity-management-section",
+  title: "Identity Management",
+  description: "What your users need for authentication and user management",
+  cards: [
+    {
+      title: "Implement User Login",
+      description: "Add email/password authentication to your application",
+      language: "auth",
+      languageLogoAlt: "Authentication icon",
+      docs: "/docs/kratos/concepts/credentials/username-email-password",
+    },
+    {
+      title: "Enable Passwordless Authentication",
+      description: "Let users sign in without passwords via email links or SMS",
+      language: "magic",
+      languageLogoAlt: "Magic link icon",
+      docs: "/docs/kratos/passwordless/passwordless",
```

Comment:
Maybe separate passkeys ( https://docs-mt7nbq3jw-ory.vercel.app/docs/kratos/passwordless/passkeys ) and otp via email/sms ( https://docs-mt7nbq3jw-ory.vercel.app/docs/kratos/passwordless/one-time-code ) ?